vtex-apps/wishlist
# Correct use of messages

#### What is the purpose of this pull request?

As the title says

 #### What problem is this solving?

When you leave the messages used in a separate file, the following error is displayed:

```warn:
React builder could not parse automatically all messages in your code. Thus, pages containing this app may run MUCH SLOWER than average.
You can fix this by using the FormattedMessage component from react-intl in the following way:

 <FormattedMessage id="my.static.message" defaultMessage="My Static Content" />

If you can not use this component or need a dynamic message, please use the "defineMessages" function, also provided by react-intl.
Messages causing this conflict:

       wishlist-new
       wishlist-add-button
       wishlist-list-name-label
       wishlist-list-name-placeholder
       wishlist-is-public
       wishlist-is-public-hint
       wishlist-add-to-list
       wishlist-my-lists
       wishlist-buy-all
       wishlist-apply
       wishlist-add-product-fail
       wishlist-see-lists
       wishlist-product-added-to-list
       wishlist-no-list-created
       wishlist-default-list-name
       wishlist-option-delete
       wishlist-dialog-confirm
       wishlist-dialog-cancel
       wishlist-delete-confirmation-message
       wishlist-option-configuration
       wishlist-save
       wishlist-buy-items
       wishlist-quantity-selected-items
       wishlist-total
       wishlist-list-empty
       wishlist-add-items
       wishlist-list-updated
       wishlist-quantity-of-items
       wishlist-add-to-cart-success
       wishlist-add-to-cart-fail
       wishlist-quantity-products
       wishlist-copy-to
       wishlist-copied
       wishlist-see
       wishlist-list-created
```

So, each message must be in the file that will use it.

 #### How should this be manually tested?

[Workspace](https://thalyta--storecomponents.myvtex.com/)

1. Checkout branch `feature/correct-use-of-messages`
2. Run `vtex link`
The error should not be displayed

 #### Types of changes
